exercises/saddle-points/tests/saddle-points.rs

```diff
@@ -0,0 +1,68 @@
+extern crate saddle_points;
+
+use saddle_points::*;
+
+#[test]
+fn identify_single_saddle_point() {
+    let input: Vec<Vec<u64>> = vec![vec![9, 8, 7],
+                                    vec![5, 3, 2],
+                                    vec![6, 6, 7]];
+    assert_eq!(vec![(1,0)], find_saddle_points(input));
+}
+
+
+#[test]
+#[ignore]
+fn identify_empty_matrix() {
+    let input: Vec<Vec<u64>> = vec![vec![]];
+    let expected: Vec<(u64, u64)> = Vec::new();
+
+    assert_eq!(expected, find_saddle_points(input));
+}
+
+#[test]
+#[ignore]
+fn identify_missing_saddle_point() {
```

Comment:
Use of "missing" in this test name implies that one should be present. Please update the name of this function to make it clear that there are expected to be none.

I believe this is based on the canonical test "Can identify lack of saddle points when there are none". If so, consider naming this test based on that name. The test names don't need to precisely map to the canonical test descriptions, but it's best when there is at least some correspondence.